styles.css

```diff
@@ -87,279 +485,279 @@
-}
+  #frameworks-1 {
+    display: flex;
+    gap: 170px;
+    align-items: center;
+  }
 
-#frameworks-2 {
-  display: flex;
-  align-items: center;
-  gap: 212px;
-}
+  #frameworks-2 {
+    display: flex;
+    align-items: center;
+    gap: 212px;
+  }
 
-#contact-container {
-  background: #6070ff;
-  border-top-left-radius: 55px;
-  width: 375px;
-  background-image: url(/media/contact\ form\ shapes-mobile.svg);
-  background-position: 100% 3%;
-  background-repeat: no-repeat;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  padding: 0;
-  box-sizing: content-box;
-  padding-right: 0;
-  margin-right: 0;
-}
+  #contact-container {
+    background: #6070ff;
+    border-top-left-radius: 55px;
+    width: 100%;
+    background-image: url(/media/contact\ form\ shapes-mobile.svg);
+    background-position: 100% 3%;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    box-sizing: content-box;
+    padding-right: 0;
+    margin-right: 0;
+  }
 
-#contact-me {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  color: #fff;
-  margin-top: 103px;
-  margin-bottom: 10px;
-  height: 56px;
-}
+  #contact-me {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin-top: 103px;
+    margin-bottom: 10px;
+    height: 56px;
+  }
 
-#my-contact-me {
-  width: 327px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  text-align: center;
-  color: #ebebff;
-  margin-left: 24px;
-  margin-right: 24px;
-}
+  #my-contact-me {
+    width: 327px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
 
-.my-name {
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  height: 48px;
-  width: 327px;
-}
+  .my-name {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    height: 48px;
+    width: 327px;
+  }
 
-.my-message {
-  width: 327px;
-  height: 170px;
-  border-radius: 8px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  font-family: poppins, sans-serif;
-}
+  .my-message {
+    width: 327px;
+    height: 170px;
+    border-radius: 8px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    font-family: poppins, sans-serif;
+  }
 
-.get-in-touch {
-  margin-top: 0%;
-  margin-left: 0;
-  font-weight: 500;
-  letter-spacing: 0.03em;
-  line-height: 24px;
+  .get-in-touch {
+    margin-top: 0%;
+    margin-left: 0;
+    font-weight: 500;
+    letter-spacing: 0.03em;
+    line-height: 24px;
+  }
```

Comment:
- [ ] I would advise you to center this button so that it would look like the Figma design.
- [ ] Also, kindly remove the empty space I underlined.

### Your work
![image](https://user-images.githubusercontent.com/86200356/214822826-2b099b4a-451c-483e-9f69-4aa5237856f6.png)

### Figma design
![image](https://user-images.githubusercontent.com/86200356/214823264-b5bca48a-ecd3-42c2-a1e9-1e951ba77c08.png)